.goreleaser.yml

```diff
@@ -56,8 +58,10 @@ dockers:
   image_templates:
   - 'ghcr.io/kanisterio/controller:{{ .Tag }}'
   dockerfile: 'docker/controller/Dockerfile'
+  use: buildx
   build_flag_templates:
   - "--pull"
+  - "--platform=linux/amd64"
```

Comment:
Can this be made configurable for downstream builders?

Reply:
I think it can be done. I have still not figured this out. I will spend some more time and make this reviewable.